src/extension-support/extension-manager.js

```diff
@@ -17,7 +17,8 @@ const builtinExtensions = {
     wedo2: () => require('../extensions/scratch3_wedo2'),
     music: () => require('../extensions/scratch3_music'),
     microbit: () => require('../extensions/scratch3_microbit'),
-    playspot: () => require('../extensions/scratch3_playspot'),
+    PlaySpots: () => require('../extensions/scratch3_playspot'),
+    PlaySpotsSetup: () => require('../extensions/scratch3_playspotSetup'),
```

Comment:
bruce's comment on branding doesn't apply to variable names :)  only to the visible stuff.  I'll pass this, but I'd like to fix it to use std variable naming conventions

Reply:
I will update those back.